spec/dummy/package.json

```diff
@@ -6,6 +6,8 @@
   },
   "dependencies": {
     "@fortawesome/fontawesome-free": "^6.1.1",
+    "chart.js": "^4.5.0",
```

Comment:
check with @calebtocco, not sure if this is supposed to be in here or in rad_common_js, and just to confirm, do we need both the gem and these added?